src/leaflike/bookmarks/views.clj

```diff
@@ -2,7 +2,7 @@
   (:require [hiccup.form :as f]))
 
 (defn list-all
-  [bookmarks]
+  [bookmarks num-pages]
```

Comment:
Would this be better named page-num instead of num-pages?

Reply:
This is number of pages to show the page navigation list, not the current page number.